examples/go-sdk/http-traffic-policy.mdx

```diff
@@ -11,63 +11,95 @@ import (
 func ngrokListener(ctx context.Context) (net.Listener, error) {
   return ngrok.Listen(ctx,
     config.HTTPEndpoint(
-      config.WithPolicy(
-        policy.Policy{
-          Inbound: []policy.Rule{
-            {
-              Name: "FooBarParamNotFound",
-              Expressions: []string{"'bar' in getQueryParam('foo')"},
-              Actions: []policy.Action{
-                {
-                  Type: "custom-response",
-                  Config: map[string]any{
-                    "status_code": 400,
-                    "content": "not found",
-                    "headers": {
-                      "content-type": "text/plain"
-                    }
-                  },
-                },
-              },
-            },
-            {
-              Name: "BazCookieForLargeRequests",
-              Expressions: []string{
-                "!hasReqCookie('baz')",
-                "req.content_length > 5000",
-              },
-              Actions: []policy.Action{
-                {
-                  Type: "deny",
-                },
-              },
-            },
-          },
-          Outbound: []policy.Rule{
-             {
-              Name: "LogUnsuccessfulRequests",
-              Expressions: []string{"res.status_code < 200 && res.status_code >= 300"},
-              Actions: []policy.Action{
-                {
-                  Type: "log",
-                  Config: map[string]any{
-                    "metadata": map[string]any{
-                      "hostport": "example.com:443",
-                      "success": false,
-                    },
-                  },
-                },
-              },
-            },
-          },
-        },
-      ),
+      config.WithPolicyString(getPolicyFromFile()),
```

Comment:
should this be `config.WithTrafficPolicy()`?

Reply:
uh

Reply:
so I might've done something very dumb here

Reply:
ok yeah I 100% just updated `WithPolicyString` to the new format instead of making a new `WithTrafficPolicy` 😬 

I can have a PR up tomorrow for the new function, then update the docs

Reply:
it will basically just alias `WithPolicyString` so should be quick